usagov/usagov-2021
# USAGOV-1387 use state field

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1387

## Description

Arlene pointed out that the states already had the abbreviations in their address field. This removes the extra field and takes the abbreviation from the already existing "State" field.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Import config
- On a state page click edit, scroll down, make sure the field abbreviation is no longer there. 
- Go to /state-governments and /es/gobiernos-estatales
- On those pages in the drop down use the search as you type feature. Test that
-- typing the beginning of a state name returns the expected result
-- typing the abbreviation of a state (that you put an abbreviation in) returns the expected result
-- typing the beginning of the second word of a state (i.e "Ri" for Puerto Rico) returns the expected result
-- typing the middle of a state name (i.e "aba" for Alabama) does not return that state

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
